microcluster/app.go

```diff
@@ -117,7 +115,7 @@ func (m *MicroCluster) Status() (*internalTypes.Server, error) {
 
 // Ready waits for the daemon to report it has finished initial setup and is ready to be bootstrapped or join an
 // existing cluster.
-func (m *MicroCluster) Ready(timeoutSeconds int) error {
+func (m *MicroCluster) Ready(ctx context.Context, timeoutSeconds int) error {
```

Comment:
Isn't the `timeoutSeconds` variable now redundant as we can potentially pass a `context.WithTimeout()`?

Reply:
yeah this makes sense

Reply:
yes it is, same as in other methods like `NewCluster` and `JoinCluster`. Should I remove it on all?

Reply:
OK, I've updated it and pushed the changes